modules/quant_api/src/EventSubscriber/QuantApi.php

```diff
@@ -152,9 +152,15 @@ public function onOutput(QuantEvent $event) {
     $queue = $queue_factory->get('quant_seed_worker');
 
     foreach ($media as $item) {
-      // @todo Determine local vs. remote.
       // @todo Configurable to disallow remote files.
       // @todo Strip base domain.
+      // Do not include external items.
+      $hostname = $config->get('host_domain') ?: $_SERVER['SERVER_NAME'];
+      $check_url = parse_url($item['original_path']);
+      if (isset($check_url['host']) && $check_url['host'] !== $hostname) {
```

Comment:
Can we move this to a Helper class? It would be easier to write automated tests for this type of function if it were in its own class.

Reply:
Sounds good, will do